export const signIn = async (req, res) => {
  try {
    
  } catch (error) {
    res.status(400).send({ ok: false, msg: error })
  }
}

export const signUp = async (req, res) => {
  try {
    
  } catch (error) {
    res.status(400).send({ ok: false, msg: error })
  }
}

